packages/receiver-push-draw-id/src/handler.ts

```diff
@@ -0,0 +1,53 @@
+import { Relayer } from 'defender-relay-client';
+import { DefenderRelayProvider, DefenderRelaySigner } from 'defender-relay-client/lib/ethers';
+import {
+  receiverPushDrawId,
+  getContracts,
+  getProviderUrlOptions,
+  isMainnet,
+  isTestnet,
+  Secrets,
+} from '@pooltogether/v4-autotasks-library';
+import { mainnet, testnet } from '@pooltogether/v4-pool-data';
+
+const getBeaconChainConfig = (chainId: number, secrets: Secrets) => {
+  if (isMainnet(chainId)) {
+    return getProviderUrlOptions(1, secrets);
+  } else if (isTestnet(chainId)) {
+    return getProviderUrlOptions(5, secrets);
+  } else {
+    throw new Error(`getConfig: Unsupported network ${chainId}`);
+  }
+};
+
+export async function handler(event: any) {
```

Comment:
This autotask won't be needed since the draw will be either computed on chain or bridged from Mainnet.
So on Mainnet or L2, we can always retrieve the draw from the DrawBuffer.